docs/scenario_editor/roads.md

```diff
@@ -0,0 +1,51 @@
+## Creating the Road Network
+
+### Roads
+The monoDrive Scenario Editor provides a tool for creating road networks that vehicles in the simulation
```

Comment:
What is the tool's name? Maybe revise to scenario editor provides this _specific_ tool that allows a user to create new road networks.